---
depth_lib/Object object
float32 radius
